ksql-common/src/main/java/io/confluent/ksql/schema/ksql/FormatOptions.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.schema.ksql;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.function.Predicate;
+
+public final class FormatOptions {
```

Comment:
Why do we need this class? Why not just pass a predicate in its place?

Reply:
This is more extensible. I'm expecting more options along the way.